qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -476,34 +499,70 @@ def export_to_file(
-            `Q-CTRL Control Data Format
-            <https://docs.q-ctrl.com/wiki/output-data-formats#q-ctrl-hardware>` _.
+            Specified file format for saving the control. Defaults to 'Q-CTRL expanded'. Currently
+            does not support any other format. For detail of the Q-CTRL expanded format, see below.
         file_type : str, optional
-            One of 'CSV' or 'JSON'; defaults to 'CSV'.
+            One of 'CSV' or 'JSON'. Defaults to 'CSV'.
         coordinates : str, optional
-            Indicates the co-ordinate system requested. Must be one of
-            'cylindrical', 'cartesian'; defaults to 'cylindrical'
-
-        References
-        ----------
-        `Q-CTRL Control Data Format
-        <https://docs.q-ctrl.com/wiki/output-data-formats#q-ctrl-hardware>` _.
+            The coordinate system in which to save the control. Must be 'cylindrical' or
+            'cartesian'. Defaults to 'cylindrical'.
 
         Raises
         ------
         ArgumentsValueError
             Raised if some of the parameters are invalid.
+
+        Notes
+        -----
+        The Q-CTRL expanded format is designed for direct integration of control solutions into
+        experimental hardware. The format represents controls as vectors defined for the relevant
+        operators sampled in time (corresponding to the segmentation of the Rabi rate, azimuthal
+        angle, and detuning).
+
+        The exact data format depends on the file type and coordinate system. In all cases, the data
+        contain four lists of real floating point numbers. Each list has the same length, and the
+        :math:`n`'th element of each list describes the :math:`n`'th segment of the driven control.
+
+        For Cartesian coordinates, the four lists are X-amplitude, Y-amplitude, detuning, and
+        duration. The maximum Rabi rate is also included in the data, and the X-amplitude and
+        Y-amplitude are normalized to that maximum Rabi rate.
+
+        For cylndrical coordinates, the four lists are Rabi rate, azimuthal angle, detuning, and
+        duration. The maximum Rabi rate is also included in the data, and the the Rabi rate is
+        normalized to that maximum Rabi rate.
+
+        For CSV, the data are formatted as five columns, with one row of titles, followed by
+        :math:`N` rows of data. The first four columns contain the relevant Cartesian or cylindrical
+        data. The fifth column contains the maximum Rabi rate, and has the same value in each row.
+
+        For JSON, the data are formatted as a single object (dictionary) with four array fields, a
+        "maximum_rabi_rate" field giving the maximum Rabi rate, and optionally a "name" field giving
+        the `name` of the control.
+
+        For example, the CSV cylindrical representation of a control with two segments would be::
+
+            rabi_rate,azimuthal_angle,detuning,duration,maximum_rabi_rate
+            0.8,1.57,3000000.,0.000001,10000000
```

Comment:
This maximum Rabi rate is different the row below. Is there one zero too many here, perhaps?

Reply:
Damn it! Thanks for catching :S I went with adding an extra zero below.

Reply:
In that case, you may want to add an extra zero in the JSON representation as well

Reply:
Haha argh, how many times can I get zeros wrong? Fixed now.